.travis.yml

```diff
@@ -52,6 +52,12 @@ jobs:
 
 script: ./scripts/runTestsOnTravis.sh $TEST_SUITE
 
+# Install Syft which is used by GoReleaser in the deploy step.
+# Cosign is an optional dependency to verify the Syft binary.
+before_deploy:
+  - curl -o /usr/local/bin/cosign -L https://github.com/sigstore/cosign/releases/latest/download/cosign-linux-amd64 && chmod +x /usr/local/bin/cosign
```

Comment:
@bruth should versions of these tools be pinned to specific version?
https://reproducible-builds.org/docs/commandments/

Reply:
Cosign I am not worried about, but I will add a tag to the install script for Syft.

Reply:
As a followup, Syft is not used for the build itself, only for the SBOM files.